src/rulesets/Base/mapreduce.jl

```diff
@@ -2,6 +2,9 @@
 ##### `sum(x)`
 #####
 
+function frule((_, ẋ), ::typeof(sum), x::Tuple)
+    return sum(x), sum(ẋ)
+end
 function frule((_, ẋ), ::typeof(sum), x; dims=:)
     return sum(x; dims=dims), sum(ẋ; dims=dims)
```

Comment:
This is unrelated, it could be removed if anyone feels strongly. `sum((1,2,3); dims=:)` is an error.

Reply:
it is fine